course/style.css

```diff
@@ -0,0 +1,429 @@
+/*
+Theme Name: Course
+Author: Automattic
+Author URI: https://automattic.com/
+Description: Course is a flexible and modern theme for anyone wanting to share their knowledge. The theme is built with integration with Sensei LMS and is ideal for Sensei users that are creating or selling courses. Style variations with multiple font and color combinations help you craft the perfect look and feel to show off courses and content. The theme can be used without Sensei too.
+Version: 1.2.1
+Requires at least: 6.1
+Tested up to: 6.1
+Requires PHP: 7.2
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Text Domain: course
+Tags: block-patterns, eLearning, education, full-site-editing, lms, online courses, sensei, teach, translation-ready
+*/
+
+/*
+ * Font smoothing
+ */
+ body {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+}
+
+/*
+ * Responsive menu container padding.
+ * This ensures the responsive container inherits the same
+ * spacing defined above. This behavior may be built into
+ * the Block Editor in the future.
+ */
+.wp-block-navigation__responsive-container.is-menu-open {
```

Comment:
This CSS can be removed now I think.

Reply:
Yup! Tested and removed in https://github.com/Automattic/themes/pull/6888/commits/1ceea814319056ae138f94ba97d0a0750808e534.